src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -329,6 +329,22 @@ public void updateAdapterRequestErrorMetric(String bidder, MetricName errorMetri
         forAdapter(bidder).request().incCounter(errorMetric);
     }
 
+    public void updateAdapterRequestDisabledBidderMetric(String bidder, Account account) {
+        forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
+        if (accountMetricsVerbosityResolver.forAccount(account)
+                .isAtLeast(AccountMetricsVerbosityLevel.detailed)) {
+            forAccount(account.getId()).adapter().forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
+        }
+    }
+
+    public void updateAdapterRequestUnknownBidderMetric(String bidder, Account account) {
+        forAdapter(bidder).request().incCounter(MetricName.unknown_bidder);
```

Comment:
```suggestion
    public void incrementAdapterRequestDisabledBidderMetric(String bidder, Account account) {
        forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
        if (accountMetricsVerbosityResolver.forAccount(account)
                .isAtLeast(AccountMetricsVerbosityLevel.detailed)) {
            forAccount(account.getId()).adapter().forAdapter(bidder).request().incCounter(MetricName.disabled_bidder);
        }
    }

    public void incrementAdapterRequestUnknownBidderMetric(String bidder, Account account) {
        forAdapter(bidder).request().incCounter(MetricName.unknown_bidder);
```

I would name the methods `incrementXYZ` - update is very unspecific and can mean a lot of this.